docs/program-setup/serial/serial_communication.md

```diff
@@ -6,86 +6,85 @@ The Arm Mbed microcontroller on your board can communicate with a host PC over t
+### Opening a serial terminal after compilation
 
-### Installing an application for Windows
+You can use one command to compile an application, flash it onto a board and open a serial terminal:
 
-There are many terminal applications for Windows, including:
+```
+mbed compile -t <TOOLCHAIN> -m <TARGET> --flash --sterm
+```
 
-- [CoolTerm](http://freeware.the-meiers.org/) - this is the application we use in this example. We use it often because it usually "just works".
-- [Tera Term](http://sourceforge.jp/projects/ttssh2/files).
-- [PuTTY](http://www.chiark.greenend.org.uk/~sgtatham/putty/).
-- Some Windows PCs come with **Hyperterminal** installed.
+When the board is flashed, it starts printing to the terminal (`Hello World!`, in this example).
 
-### Configuring the connection
+To close the serial terminal, enter <kbd>Ctrl + C</kbd>.
 
-1. Plug in your Mbed board.
-1. Open CoolTerm.
-1. Click **Connect**. This opens up an 8-n-1 9600 baud connection to the first available serial port. If you have more than one board plugged in, you may need to change the port under **Options > Serial Port > Port**.
+<div style="background-color:#F3F3F3; text-align:left; vertical-align: middle; padding:15px 30px;">  <b>Note:</b>
+<br/>- If your application uses a baud rate other than 9600, specify it with `-b <BAUDRATE>` in the command above.
+<br/>- This method only works for a single board. To work with multiple boards, open a serial terminal manually as described below.</div>
 
-Check your connection parameters:
+### Manually opening a serial terminal
 
-1. Select **Options > Serial Port**.
-1. You should see 9600 baud, 8 bits, 1 stop bit, no parity (9600-8-N-1).
-1. If you do not see your board, click **Re-Scan Peripherals**.
+You can open a serial terminal manually, which is useful when the board is already flashed with the desired application (i.e. one built with the [online compiler](../quick-start/build-with-the-online-compiler.html)).
 
-Your terminal program is now configured and connected.
+When a single board is connected, run:
 
-## Using terminal applications on Linux
+```
+mbed sterm
+```
 
-CoolTerm should work under Linux. If for some reason it doesn't, you can try one of the following:
+`mbed sterm` starts message printing.
 
-- [Minicom](https://help.ubuntu.com/community/Minicom).
-- [GNU Screen](https://www.gnu.org/software/screen/manual/screen.html).
+To close the serial terminal, enter <kbd>Ctrl + C</kbd>.
 
-## Minimal Printf
+<div style="background-color:#F3F3F3; text-align:left; vertical-align: middle; padding:15px 30px;">  <b>Note:</b>
+<br/>If your application uses a baud rate other than 9600, specify it with `-b <BAUDRATE>` when opening the serial terminal.
+<br/>If you have multiple boards connected:
+1. Run `mbedls` to find the port of the board you want to use.
+1. Run `mbed sterm -p <PORT>`.
```

Comment:
Between `<div>` and `</div>` formatting doesn't seem to work - `<BAUDRATE>` isn't shown, for example.